projects/components/src/search-box/search-box.component.scss

```diff
@@ -31,7 +31,7 @@
     }
   }
 
-  &.with-search-history {
+  &.collapsable {
```

Comment:
if we rename this, does the enable search history route lose any of its styling? if not, should we just remove L41 in the component file?

Reply:
+1 we should have a separate style for this particular class or remove the other line if is useless right now.

Reply:
`.with-search-history` doesn't have any styling rules specific to search history and the search history box is not affected by this change.

All the styling rules in `.with-search-history` is only related to the collapsing logic.

Reply:
Also the class is not being used anywhere so we can remove it I think

Reply:
So should we remove the `.with-search-history` class altogether?

Reply:
if its not used, lets remove it :)